tracking/ForwardReleases.md

```diff
@@ -0,0 +1,52 @@
+# Zowe Forward Release Strategy
+
+The Zowe project and its sub-projects may have a new major release version that is ready for user validation, but not ready to replace the prior LTS version posted on zowe.org. 
+
+The zowe.org website should present this new version to the user as "Current", while the previously released major version (LTS) should be selected and presented by default.
+
+## User Experience
+
+Sample flow:
+
+On the zowe.org/download page -
+
+-----------------------------------------------
+
+ **LTS Zowe v1.x**   |  Current Zowe v2.x            
+
+ - The latest version of Zowe v1.x is: v1.2.0.  
+ - (Optional) The community support period for Zowe v1.x lasts until: <>. Commercial support periods may extend beyond this.  
+
+ ...Regular download flow / How to get started flow
+
+-----------------------------------------------
+
+ LTS Zowe v1.x   |  **Current Zowe v2.x**  
+
+ - The latest version of Zowe v2.x is: v2.0.0-alpha.1.  
+ - (Optional) The community support period for Zowe v2.x lasts until ..time period.. after it is officially released. Commercial support periods may extend beyond this.  
+
```

Comment:
I assume that the example is not assuming support for an **alpha** version

Reply:
Matt - correct - I think support here would be closer to "YMMV". The team would be actively working on the codebase and will probably have eyes on any new issues opened - but will not commit to a fix delivery date. I will adjust that comment.

I also presented the version tag as "2.0.0-alpha.1" to try and capture the goal around presenting new code that is not ready for a proper release, but more reliable than a Nightly Build. In this case, 2.0.0-alpha.1 may just be a nightly build we identify, test and present to users...